src/util/schema/BitmapSchema.ts

```diff
@@ -0,0 +1,63 @@
+/**
+ * @license
+ * Copyright 2022 Project CHIP Authors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Schema } from "./Schema";
+
+const enum BitRangeType {
+    Flag,
+    Enum,
+}
+
+/** Defines the bit position of a boolean flag. */
+export interface FlagBit { type: BitRangeType.Flag, offset: number }
+export const FlagBit = (offset: number) => ({ type: BitRangeType.Flag, offset }) as FlagBit;
```

Comment:
Rename `FlagBit` to `BitFlag`?  I like grouping all BitmapSchema subtypes under the prefix `Bit`.

Reply:
Done